### PR TITLE
#83 Feature- implement notification panel

### PR DESCRIPTION
**Pull request by Shumba250**
#### What does this PR do?
The notifications panel displays a user's notifications and prompts them to take action when they interact with notification icon
#### Description of Task to be completed

- The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications
#### How should this be manually tested?
clone the repo using git clone repo's URL
change branch git ft- implement notification panel
create .env file and paste variables from .env.example
run npm run dev
login as a vendor
You should be redirected to the dashboard
click on the notification icon and you should be able to see the notification

### CODE REVIEW

**Comment by Shumba250** [comment]
[Implement notifications pane](https://trello.com/c/6O2pHIzb/83-implement-notifications-pane)

**Review by sostene12 — APPROVED** [review]
Screenshot 2023-05-15 at 17 50 04
the notifications are working, when i make some action, but the style of products are not verywell, consider improving it

**Review by patsicko — APPROVED** [review]

File: vite.config.js.timestamp-1684135706373-7f1bd4242e6e6.mjs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
...cmVwb3J0ZXI6IFsndGV4dCcsICdqc29uJywgJ2h0bWwnXSxcblx0XHR9LFxuXHR9LFxufSk7XG4iXSwKICAibWFwcGluZ3MiOiAiO0FBQW1XLFNBQVMsb0JBQW9CO0FBQ2hZLE9BQU8sV0FBVztBQUNsQixPQUFPLFFBQVE7QUFDZixTQUFTLGVBQWU7QUFIeEIsSUFBTSxtQ0FBbUM7QUFLekMsSUFBTyxzQkFBUSxhQUFhO0FBQUEsRUFDM0IsU0FBUyxDQUFDLE1BQU0sQ0FBQztBQUFBLEVBQ2pCLE9BQU87QUFBQSxJQUNOLGVBQWU7QUFBQSxNQUNkLE9BQU87QUFBQSxRQUNOLE1BQU0sUUFBUSxrQ0FBVyxZQUFZO0FBQUEsTUFDdEM7QUFBQSxNQUNBLFFBQVE7QUFBQSxRQUNQLFNBQVM7QUFBQSxVQUNSLE9BQU87QUFBQSxRQUNSO0FBQUEsTUFDRDtBQUFBLElBQ0Q7QUFBQSxFQUNEO0FBQUEsRUFDQSxTQUFTO0FBQUEsSUFDUixRQUFRO0FBQUEsSUFDUixTQUFTO0FBQUEsSUFDVCxTQUFTLENBQUM7QUFBQSxFQUNYO0FBQUEsRUFDQSxjQUFjO0FBQUEsSUFDYixnQkFBZ0I7QUFBQSxNQUNmLFNBQVM7QUFBQSxRQUNSO0FBQUEsVUFDQyxNQUFNO0FBQUEsVUFDTixNQUFNLE9BQU87QUFDWixrQkFBTSxPQUFPLEVBQUUsUUFBUSxlQUFlLEdBQUcsT0FBTyxVQUFVO0FBQUEsY0FDekQsUUFBUTtBQUFBLGNBQ1IsVUFBVSxNQUFNLEdBQUcsU0FBUyxLQUFLLE1BQU0sTUFBTTtBQUFBLFlBQzlDLEVBQUU7QUFBQSxVQUNIO0FBQUEsUUFDRDtBQUFBLE1BQ0Q7QUFBQSxJQUNEO0FBQUEsRUFDRDtBQUFBLEVBQ0EsTUFBTTtBQUFBLElBQ0wsU0FBUztBQUFBLElBQ1QsYUFBYTtBQUFBLElBQ2IsWUFBWTtBQUFBLElBQ1osVUFBVTtBQUFBLE1BQ1QsVUFBVTtBQUFBLE1BQ1YsVUFBVSxDQUFDLFFBQVEsUUFBUSxNQUFNO0FBQUEsSUFDbEM7QUFBQSxFQUNEO0FBQ0QsQ0FBQzsiLAogICJuYW1lcyI6IFtdCn0K
```
Comment (wakaflorien) [inline vite.config.js.timestamp-1684135706373-7f1bd4242e6e6.mjs]: - check the vite config file.js the commented section

File: src/main.jsx [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
 import App from './App';
 import './index.css';
 import store from './redux/store';
+import { ContextProvider } from './contexts/ContextProvider';
+
+registerLicense(
+	'Mgo+DSMBaFt+QHJqVk1mQ1JbdF9AXnNNdFB3T2Zebz4Nf1dGYl9RQXZbQ1tgSHtUf0xnWQ==;Mgo+DSMBPh8sVXJ1S0R+X1pCaVxdX2NLfUNyT2BYdVt3ZDU7a15RRnVfR11nSHhQd0BrWX5ecA==;ORg4AjUWIQA/Gnt2VFhiQlJPcEBBQmFJfFBmQWlaflR1c0UmHVdTRHRcQlhjTX9VdURmUX9ceHQ=;MTkyNjU1NkAzMjMxMmUzMjJlMzNPYy9kbVlub2xtaHZPeTcrQTQydCtoN29wRWNWd3I4ejNFanIwWmJNMklvPQ==;MTkyNjU1N0AzMjMxMmUzMjJlMzNlS290R2VBK0lsTmt5amdldjhUcXB0TTFjbXRTOUgrTVZhUis1c3JqWFRvPQ==;NRAiBiAaIQQuGjN/V0d+Xk9HfVldXnxLflF1VWdTfF96cVJWACFaRnZdQV1mSXpScEZiXHdeeX1V;MTkyNjU1OUAzMjMxMmUzMjJlMzNVNzViT2lxRGxEa1dZeFM5N2VZUHVlMmZCZkhEcGM1MFd4eDEzM0thUVJjPQ==;MTkyNjU2MEAzMjMxMmUzMjJlMzNZWXRSR3dKQmgzaVQ5OC9RWDc4VXp0bmdFbE8xNVY5cGRYZi81RjhzSTdFPQ==;Mgo+DSMBMAY9C3t2VFhiQlJPcEBBQmFJfFBmQWlaflR1c0UmHVdTRHRcQlhjTX9VdURmUX5YdnQ=;MTkyNjU2MkAzMjMxMmUzMjJlMzNoL0NHTnlFZVBxT1NRNytPZDNuSlBFSkMwaWJQVENZbWlESmQ3c016MVJzPQ==;MTkyNjU2M0AzMjMxMmUzMjJlMzNjdFhKQ3BSa09lUVFBQWNTZjZXME14YzZDMUg1STN4dy9SY2hpY3Fla0lFPQ==;MTkyNjU2NEAzMjMxMmUzMjJlMzNVNzViT2lxRGxEa1dZeFM5N2VZUHVlMmZCZkhEcGM1MFd4eDEzM0thUVJjPQ=='
```
Comment (wakaflorien) [inline src/main.jsx]: -  ass license to .env file

**Review by wakaflorien — CHANGES_REQUESTED** [review]

**Review by wakaflorien — APPROVED** [review]

**Pull request merged**